docs/current_docs/ci/integrations/kubernetes.mdx

```diff
@@ -195,6 +195,13 @@ dagger query <<EOF
 EOF
 ```
 
+## Tested Configurations
```

Comment:
We're following a standard template for all integrations pages so for consistency with others, it would be better to not have this at the top level. Would you consider moving this new section under "How it Works" before or after the "Recommendations" section?

Reply:
sure

Reply:
done